Scripts/UnzipFile/UnzipFile_test.py

```diff
@@ -0,0 +1,27 @@
+import shutil
+
+from UnzipFile import *
+import os
+import pytest
+
+
+data_test_extract = [
+    ('testZip.yml', None),
+    ('ScanSummary.txt', None),
+    ('item.png', None),
+    ('fix_unzip.png', 'demisto'),
+]
+
+
+@pytest.mark.parametrize('file_name, password', data_test_extract)
+def test_extract(file_name, password):
```

Comment:
the test is a bit hard to read.
explain in steps what are you testing
use the template:
```
Given
- valid zip file
- empty folder _dir

When
- run extract on that zip file and export the internal content to _dir

Then
- ensure zip file content have be saved at _dir directory with the original filename
- ensure that the saved file has expected content
```